src/main/controllers/BlobViewLocationController.ts

```diff
@@ -0,0 +1,30 @@
+import {PipRequest} from '../models/request/PipRequest';
+import {Response} from 'express';
+import {cloneDeep} from 'lodash';
+import {LocationService} from '../service/locationService';
+import {PublicationService} from '../service/publicationService';
+
+const locationService = new LocationService();
+const publicationService = new PublicationService();
+export default class BlobViewLocationController {
+  public async get(req: PipRequest, res: Response): Promise<void> {
+    const list_of_locs = await locationService.fetchAllLocations(req.lng);
+    const counts = await publicationService.getCountsOfPubsPerLocation();
+    if (list_of_locs && counts) {
```

Comment:
Can we use camel case over snake case please to keep in line with the rest of the repo.

Reply:
sorted